src/test/fixtures/profiles/processed-profile.js

```diff
@@ -1357,22 +1357,26 @@ export function getProfileWithUnbalancedNativeAllocations() {
   const {
     profile,
     funcNamesDictPerThread: [funcNamesDict],
-  } = getProfileFromTextSamples(`
-    A  A     A
-    B  B     B
-    C  Fjs   Fjs
-    D  Gjs   Gjs
-    E        Hjs[lib:jQuery.js]
-             I[lib:libI.so]
-  `);
+  } = getProfileFromTextSamples(
+    // We need to take care that A, B, Gjs, Gjs has the most samples, because
+    // some tests rely on this.
```

Comment:
In tests, the call tree is displayed first with the "timings" data, so `procureInitialInterestingSelection` uses this data, and only _then_ we switch to the memory data. So I wanted to have the same expanded nodes than we had before, so that the unrelated snapshots don't change.

Reply:
Thought (no action needed): I feel like this `procureInitialInterestingSelection` has many failure modes, I wonder if it would be better to hide this behind a selector and improve it for better default results? I feel like it frequently fails to run when selecting different threads, and I get a collapsed call tree, which isn't great UX.

Reply:
Yeah, it's very common we get a collapsed call tree, I think this is especially when changing from JS implementation to "all stacks", or when popping a transform (possibly ranges too? Not sure).